groovy/groovy.debug/src/org/netbeans/modules/groovy/debug/GroovyBreakpointAnnotationListener.java

```diff
@@ -71,16 +71,22 @@ public void propertyChange (PropertyChangeEvent e) {
     @Override
     public void breakpointAdded (Breakpoint b) {
         if (b instanceof LineBreakpoint) {
-            ((LineBreakpoint) b).addPropertyChangeListener (this);
-            annotate ((LineBreakpoint) b);
+            LineBreakpoint lb = (LineBreakpoint) b;
```

Comment:
Only act on breakpoint which has Groovy stratum. Java files will be handled by the Java JPDA breakpoint handler.